pkg/template/naming_strategies.go

```diff
@@ -0,0 +1,9 @@
+package template
+
+import "strings"
+
+var tplFuncs = map[string]interface{}{
+	"upper": strings.ToUpper,
+	"title": strings.Title,
+	"lower": strings.ToLower,
+}
```

Comment:
I would copy this content into `naming.go`

Reply:
done